### DISCUSSION TITLE
Pixi v8 docs?

### DISCUSSION THREAD

**Discussion by Nantris** [pixijs/pixijs#10132] [General]
I haven't been able to find them. Are they easily accessible somewhere? I assume so since a lot of people are using v8.

**Comment by Nantris** [comment]
I see they're mentioned on the releases themselves. https://pixijs.download/v8.0.0-rc.5/docs/index.html

**Comment by dschumak** [comment]
Thank you! It took me longer than it should have to find this. +++

**Comment by RPGillespie6** [comment]
The v8 docs have a lot of broken links and missing/inconsistent info. Where should we report issues of that nature?

For example:
1. [Application](https://pixijs.download/release/docs/app.Application.html) links to a broken [Renderer](https://pixijs.download/release/docs/global.html#Renderer) page
2. [Playground](https://pixijs.com/8.x/playground) shows `background` being an available option supplied to `Application.init` but I don't see that in [ApplicationOptions](https://pixijs.download/release/docs/app.ApplicationOptions.html).
3. [Texture](https://pixijs.download/release/docs/rendering.Texture.html) doesn't link to `TextureOptions`

**Reply by bigtimebuddy** [reply]
This is great for an issue! Would you mind filing a bug? Thanks for pointing these out. Likely it is not intentional but probably an oversight.

**Reply by RPGillespie6** [reply]
No problem: https://github.com/pixijs/pixijs/issues/10663
